suite-native/module-connect-popup/src/components/ConnectPopupDebugOptions.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { ScrollView } from 'react-native';
+
+import * as Linking from 'expo-linking';
+
+import { BottomSheet, Button } from '@suite-native/atoms';
+
+import { buildURL } from '../utils/buildURL';
+
+type ConnectPopupDebugOptionsProps = React.PropsWithChildren<{
+    showDebug: boolean;
+    setShowDebug: (show: boolean) => void;
+}>;
+
+export const ConnectPopupDebugOptions = ({
+    showDebug,
+    setShowDebug,
+    children,
+}: ConnectPopupDebugOptionsProps) => {
+    return (
+        <BottomSheet
+            isVisible={showDebug}
+            onClose={() => {
+                setShowDebug(false);
+            }}
+        >
+            <ScrollView>
```

Comment:
There is already ScrollView inside of BottomSheet this one shoudn't be necessary and will probably break some gestures etc.